### PR TITLE
Version 2 changes

### PR DESCRIPTION
**Pull request by kd4001**

### CODE REVIEW

File: src/components/generic/list/CheckBoxListItem.re [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    (
+      ~itemText: React.element,
+      ~ischecked: bool,
+      ~checkboxClassName: string="",
+      ~itemTextClassName: string="",
+    ) => {
+  <>
+    <MaterialUi.ListItemIcon
+      classes={MaterialUi.ListItemIcon.Classes.make(~root="root " ++ checkboxClassName, ())}>
+      <MaterialUi.Checkbox edge=MaterialUi.Checkbox.Edge.start checked=ischecked color=`Primary />
```
Comment (Prateek13727) [inline src/components/generic/list/CheckBoxListItem.re:12]: can you verify if color=`Primary still uses polymorphic variant ?

File: src/components/generic/menu-list/MenuListV2.re [hunk 2]
```diff
@@ -52,10 +52,10 @@ let make =
       content={popoverState.view}
```
Comment (Prateek13727) [inline src/components/generic/menu-list/MenuListV2.re:52]: Remove MenuList component if it is not being used anywhere and rename this to MenuList

File: src/components/generic/list/CheckBoxListItem.re (hunk 1 above)
Comment (kd4001) [reply src/components/generic/list/CheckBoxListItem.re:12]: Still a polymorphic variant as of now.

File: package.json [hunk 3]
```diff
@@ -24,24 +24,26 @@
...
     "bs-webapi": "^0.17.1",
     "decco": "^1.1.1",
-    "papaparse": "5.1.0"
+    "papaparse": "5.1.0",
+    "moment": "^2.24.0",
+    "@date-io/core": "^1.3.6"
   },
   "devDependencies": {
-    "bs-platform": "^7.0.1",
+    "bs-platform": "^8.2.0",
```
Comment (Prateek13727) [inline package.json:41]: please upgrade-reason-react 0.9.1

Comment (Prateek13727) [reply package.json:41]: also verify the same in Spot-CRM

**Pull request closed without merging**